fixtures/custom-rules/rules/CUSTOM-4/fixtures/test.tf

```diff
@@ -0,0 +1,9 @@
+# Configure the AWS Provider
```

Comment:
For future reference, we're keeping the original TF file and the binary so we can recreate this scenario if needed